### DISCUSSION TITLE
Nix flake version

### DISCUSSION THREAD

**Discussion by Dregorio1** [hyprwm/Hyprland#5733] [Q&A] [answered]
Hello, I have in my flake.nix:
```
inputs.hyprland.url = "github:hyprwm/Hyprland/v0.39.1";
```
But when I run in terminal:
```
$ hyprctl version
```
I have got 0.32.3. Update of my flake failed or version numeration is wrong?

**Comment by fufexan** [comment, marked as answer]
0.32.3 is the version that NixOS Stable has. Are you sure you're passing the flake package to both the NixOS and HM modules (if you use HM as well)?

**Reply by Dregorio1** [reply]
@fufexan Thank you, I am moron and I used the version from chanel(?). I passed as you said and works. Thank you and sorry for stupid question
